src/Analysis/Ast/Impl/Types/ParameterInfo.cs

```diff
@@ -29,7 +29,7 @@ public ParameterInfo(PythonAst ast, Parameter p, IPythonType type, IMember defau
         }
 
         public ParameterInfo(string name, IPythonType type, ParameterKind? kind, IMember defaultValue) {
-            Name = name ?? throw new ArgumentNullException(nameof(name));
+            Name = name ?? "*"; // ?? throw new ArgumentNullException(nameof(name));
```

Comment:
Is it temporary?

Reply:
The problem is that sometimes argument does not have name. This happens to `*` arguments since parser strips the `*`. So I'd like to have at least something.